studio/components/interfaces/Database/Wrappers/Wrappers.constants.ts

```diff
@@ -676,4 +676,78 @@ export const WRAPPERS: WrapperMeta[] = [
       },
     ],
   },
+  {
+    name: 'bigquery_wrapper',
+    handlerName: 'big_query_fdw_handler',
+    validatorName: 'big_query_fdw_validator',
+    icon: `${BASE_PATH}/img/icons/bigquery-icon.svg`,
+    extensionName: 'BigQueryFdw',
+    label: 'BigQuery',
+    docsUrl: 'https://supabase.github.io/wrappers/bigquery/',
+    server: {
+      options: [
+        {
+          name: 'sa_key_id',
+          label: 'Service Account Key',
+          required: true,
+          encrypted: true,
+          hidden: true,
+        },
+        {
+          name: 'project_id',
+          label: 'Project ID',
+          required: true,
+          encrypted: false,
+          hidden: false,
+        },
+        {
+          name: 'dataset_id',
+          label: 'Dataset ID',
+          required: true,
```

Comment:
@burmecia is `dataset_id` always required?

Reply:
yes, it is required for server option, same for `project_id` as well.